plugins/ui/src/deephaven/ui/components/radio_group.py

```diff
@@ -1,14 +1,174 @@
+from __future__ import annotations
+from typing import Any, Callable
+from .types import (
+    # Events
+    FocusEventCallable,
+    # Layout
+    AlignSelf,
+    CSSProperties,
+    DimensionValue,
+    JustifySelf,
+    LayoutFlex,
+    Position,
+    LabelPosition,
+    Align,
+    # Validation
+    NecessityIndicator,
+    Orientation,
+    ValidationBehavior,
+)
 from .basic import component_element
+from ..elements import Element
+from .._internal.utils import create_props
+
 
 # TODO: Add pydocs #514
-def radio_group(*children, **props):
+def radio_group(
+    *children,
+    is_emphasized: bool | None = None,
+    orientation: Orientation = "vertical",
+    value: str | None = None,
+    default_value: str | None = None,
+    is_disabled: bool | None = None,
+    is_read_only: bool | None = None,
+    name: str | None = None,
+    is_required: bool | None = None,
+    is_invalid: bool | None = None,
+    validation_behaviour: ValidationBehavior | None = None,
```

Comment:
Your Canadian is showing:
```suggestion
    validation_behavior: ValidationBehavior | None = None,
```